smarts/core/utils/traffic_history_service.py

```diff
@@ -162,7 +164,21 @@ def fetch_history_at_timestep(self, timestep: str):
         return {}
 
     @staticmethod
-    def fetch_agent_missions(history_file_path):
+    def fetch_agent_missions(history_file_path: str, scenario_root_path: str):
+        assert os.path.isdir(scenario_root_path)
+        history_mission_filepath = os.path.join(
+            scenario_root_path, "temp_vehicle_mission.pkl"
```

Comment:
nit: I think this file does not fully describe what it is for. Perhaps `history_missions.pkl`?